ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.9-presubmits.yaml

```diff
@@ -412,6 +412,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )lint,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^release-4\.9$
+    - ^release-4\.9-
+    cluster: build02
```

Comment:
@mansikulkarni96 I think this should be vsphere and not build02.  This would possibly explain why neither the VMC or IBM vCenters are reachable.